.github/workflows/build-and-test-pr.yml

```
@@ -147,19 +179,19 @@ jobs:
+      - name: "Cache zookeeper binary archive"
+        uses: actions/cache@v3
         id: "cache-zookeeper"
         with:
           path: ${{ github.workspace }}/.tmp/zookeeper
           key: zookeeper-${{ runner.os }}-${{ env.ZOOKEEPER_VERSION }}
           restore-keys: |
             zookeeper-${{ runner.os }}-${{ env.ZOOKEEPER_VERSION }}
       - name: "Set up msys2 if necessary"
-        if: ${{ startsWith( matrix.os, 'windows') && steps.cache-zookeeper.outputs.cache-hit != 'true' }}
         uses: msys2/setup-msys2@v2
+        if: ${{ startsWith( matrix.os, 'windows') && steps.cache-zookeeper.outputs.cache-hit != 'true' }}
```

actionlint (1.7.4)

property "os" is not defined in object type {}

(expression)